src/dnddiceroller/DiceTower.java

```diff
@@ -15,8 +15,9 @@
  * @author Paul Scarrone
  */
 public class DiceTower {
-  final int PANEL_COUNT = 3;
-  List<Die> dice;
+  final private int PANEL_COUNT = 3;
```

Comment:
While this is not wrong we usually do `<Access Modifier> <Other Modifiers> <type> <variable name>` as the pattern here